sentry/src/main/java/io/sentry/metrics/IMetricsHub.java

```diff
@@ -0,0 +1,24 @@
+package io.sentry.metrics;
+
+import java.util.Map;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public interface IMetricsHub {
+  /** Captures one or more metrics to be sent to Sentry. */
+  void captureMetrics(final @NotNull EncodedMetrics metrics);
```

Comment:
would it make sense to add this to `IHub` interface directly?

Reply:
I wouldn't, at least it makes it harder to test and we need to change it again once we merge scopes and hubs